src/modules/flow/iio/iio.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "iio",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD 3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "input/hw",
+      "description": "IIO based gyroscope input node. TODO explain the wonderful world of triggers, buffers and buffer sized 0 and -1.",
+      "in_ports": [
+        {
+          "data_type": "any",
+          "description": "Packets sent in here will trigger a gyroscope reading and produce packets on each of the the output ports.",
+          "name": "TICK",
+          "methods": {
+            "process": "gyroscope_tick"
+          }
+        }
+      ],
+      "methods": {
+        "close": "gyroscope_close",
+        "open": "gyroscope_open"
+      },
+      "name": "iio/gyroscope",
+      "options": {
+       "members": [
+         {
+           "data_type": "int",
```

Comment:
is this fixed? If it may change according to some setup, ie if user adds more devices, then could we make this a string and accept an integer (checked with strtoul()) and also resolve the number using libudev if it's available? This way we could be more resilient to configuration changes.